Testing/functional/tests/cypress/integration/e2e/timeline/bcCancerScreening.js

```diff
@@ -0,0 +1,76 @@
+const { AuthMethod } = require("../../../support/constants");
+
+describe("Diagnostic Imaging", () => {
+    beforeEach(() => {
+        cy.configureSettings({
+            datasets: [
+                {
+                    name: "bcCancerScreening",
+                    enabled: true,
+                },
+            ],
+        });
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+    });
+
+    it("Validate Card Details", () => {
+        cy.get("[data-testid=timelineCard")
+            .first()
+            .within(() => {
+                cy.get("[data-testid=bccancerscreeningTitle]")
+                    .should("be.visible")
+                    .click({ force: true });
+                cy.get(
+                    "[data-testid=bc-cancer-screening-download-button]"
+                ).should("be.visible");
+            });
+    });
+
+    it("Validate file download", () => {
+        cy.get("[data-testid=timelineCard")
+            .first()
+            .within(() => {
+                cy.get("[data-testid=bccancerscreeningTitle]")
+                    .should("be.visible")
+                    .click({ force: true });
+                cy.get(
+                    "[data-testid=bc-cancer-screening-download-button]"
+                ).should("be.visible");
+                // Test for generic message modal regarding sensitive data
+                cy.get(
+                    "[data-testid=bc-cancer-screening-download-button]"
+                ).click();
+                cy.document()
+                    .find("[data-testid=generic-message-modal]")
+                    .should("be.visible");
```

Comment:
The Cypress docs suggest using `cy.root().closest()` to [temporarily escape](https://docs.cypress.io/api/commands/within#Temporarily-escape) a `within()`.

Reply:
I'll take a look I remember root not working when I tried, I found a stackoverflow answer that recommended document()